src/Host/Broker/Impl/Extensions/BasicSignInContextExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Odachi.AspNetCore.Authentication.Basic;
+
+namespace Microsoft.R.Host.Broker {
+    public static class BasicSignInContextExtensions {
+        public static bool IsSignInRequired(this BasicSignInContext context) {
+            string path = context.HttpContext.Request.Path.ToString();
+            if (path.Equals("/info/load") || path.Equals("/info/about")) {
```

Comment:
Let's make it a `static readonly HashSet` so that it's easy to add new paths as needed.

(Later, might want to look at detecting `[AllowAnonymous]` attributes instead, but for now let's keep it simple.)